ci/kokoro/docker/Dockerfile.fedora-install

```diff
@@ -40,15 +40,15 @@ RUN dnf makecache && dnf install -y \
     openssl-devel \
     pkgconfig \
     protobuf-compiler \
-    python \
+    python38 \
     shtool \
     unzip \
     wget \
     which \
     zlib-devel
 
-RUN pip install --upgrade pip
-RUN pip install httpbin flask gevent gunicorn crc32c
+RUN python3.8 -m pip install --upgrade pip
```

Comment:
Genuine question: is it better to do `python3 -m pip` or `pip3` here?

Reply:
Not all OS platforms have `pip3`. Calling `python3 -m pip` ensures the pip we are using is the one connected to that particular interpreter. So the more consistent experience across platforms and double-sureness is why I tend to do this. In theory, they are equivalent.